### PR TITLE
[Scalability] Add new oncall member

### PR DESCRIPTION
**Pull request by jupblb**
/cc @wojtek-t
/cc @marseel 

I am a new member of sig-scalability oncall. :)

### CODE REVIEW

File: groups/groups.yaml [hunk 1]
```diff
@@ -387,6 +387,7 @@ groups:
       - janluk@google.com
       - jkaniuk@google.com
       - jprzychodzen@google.com
+      - jupblb@google.com
```
Comment (wojtek-t) [inline groups/groups.yaml:390]: Since you touch this file - can you please remove janluk@ ?

Please also add marcelzieba@google.com

Comment (jupblb) [reply groups/groups.yaml:389]: Done.

**Pull request merged**